app/contactus/page.tsx

```
@@ -0,0 +1,166 @@
+		}
+	};
+
+	useEffect(() => {
+		// Any client-only code can go here
+	}, []);
+
+	return (
+		<div>
+			<div className='flex flex-col justify-center items-center my-10'>
+				<h1 className='text-7xl font-bold mb-10'>
+					<i>Let's Start A New Project</i>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)